orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/SadStage.java

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.orca.pipeline;
+
+import com.netflix.spinnaker.orca.api.pipeline.graph.StageDefinitionBuilder;
+import com.netflix.spinnaker.orca.api.pipeline.graph.TaskNode;
+import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution;
+import com.netflix.spinnaker.orca.pipeline.tasks.SadNoOpTask;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SadStage implements StageDefinitionBuilder {
+  public static String STAGE_TYPE = "sad";
+
+  @Override
+  public void taskGraph(@NotNull StageExecution stage, @NotNull TaskNode.Builder builder) {
+    builder.withTask("sadNoOp", SadNoOpTask.class);
+  }
+}
```

Comment:
This looks like a stage for testing? Not sure I want to ship this to everyone. Can we put this in -nflx if we need it?

Reply:
removed in b98bf056e